homeassistant/components/google_generative_ai_conversation/config_flow.py

```diff
@@ -181,6 +192,7 @@ def async_get_supported_subentry_types(
         return {
             "conversation": LLMSubentryFlowHandler,
             "tts": LLMSubentryFlowHandler,
+            "ai_task_data": LLMSubentryFlowHandler,
```

Comment:
Does the AI task need a user configurable prompt?

Reply:
No. There is no prompt for AI tasks.